partition/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub(crate) static SIGNAL_STACK: Lazy<MmapMut> = Lazy::new(|| {
         .unwrap()
 });
 
-pub(crate) static SYSCALL: Lazy<RawFd> = Lazy::new(|| {
+pub(crate) static SYSCALL: Lazy<OwnedFd> = Lazy::new(|| {
```

Comment:
Can we do this in a separate (logical) commit or is it required for the update?

Reply:
yes. done